rdt/transformers/__init__.py

```diff
@@ -94,6 +94,21 @@ def get_transformer_name(transformer):
 }
 
 
+@lru_cache()
```

Comment:
@pvk-developer @amontanez24 wrote this method to address your comment regarding the `TRANSFORMERS_TO_CLASS` dict.

It feels weird to use cache with only 1 key, but I didn't want to import the dictionary itself 🤷